tests/rabota_by_tests.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+from models.parser_jobs_tut_by import RabotaByParser
+
+
+def test_connection(get_response):
+    """
+    test connection of the web site
+    """
+    assert get_response.status_code == 200
+
+
+def test_shotgun_not_found(get_query_shotgun):
+    """
+    test that there are no search results for the word "shotgun"
+    """
+    assert 'По запросу «shotgun» ничего не найдено' in get_query_shotgun.text
```

Comment:
no need to write docstrings for tests, remove

Reply:
Done